src/wp-includes/block-template-part-utils.php

```diff
@@ -0,0 +1,180 @@
+<?php
+/**
+ * Block template part functions.
+ *
+ * @package gutenberg
+ */
+
+// Define constants for supported wp_template_part_area taxonomy.
+if ( ! defined( 'WP_TEMPLATE_PART_AREA_HEADER' ) ) {
+	define( 'WP_TEMPLATE_PART_AREA_HEADER', 'header' );
+}
+if ( ! defined( 'WP_TEMPLATE_PART_AREA_FOOTER' ) ) {
+	define( 'WP_TEMPLATE_PART_AREA_FOOTER', 'footer' );
+}
+if ( ! defined( 'WP_TEMPLATE_PART_AREA_SIDEBAR' ) ) {
+	define( 'WP_TEMPLATE_PART_AREA_SIDEBAR', 'sidebar' );
+}
+if ( ! defined( 'WP_TEMPLATE_PART_AREA_UNCATEGORIZED' ) ) {
+	define( 'WP_TEMPLATE_PART_AREA_UNCATEGORIZED', 'uncategorized' );
+}
+
+/**
+ * Fixes the label of the 'wp_template_part' admin menu entry.
+ */
+function gutenberg_fix_template_part_admin_menu_entry() {
```

Comment:
I'm not sure the admin menu is one we want to include, I'll remove this for now.

Reply:
Yeah, I wasn't too sure about that myself. Makes sense to apply the same logic as we're doing with templates.